configure.ac

```diff
@@ -100,6 +100,19 @@ if test "$os_unix" = "yes"; then
 		[AC_DEFINE([HAVE_GMTIME_R], 1, [Whether gmtime_r() is available])],
 		[AC_MSG_ERROR([could not find required gmtime_r() function])])
 
+	AC_MSG_CHECKING([whether getpwuid_r() accepts 5 arguments])
+	AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <pwd.h>]],
+	                                   [[getpwuid_r(0,0,0,0,0);]])],
+					  [],
+					  [AC_COMPILE_IFELSE(
+					   [AC_LANG_PROGRAM([[#define _POSIX_PTHREAD_SEMANTICS
+#include <pwd.h>]],
+									    [[getpwuid_r(0,0,0,0,0);]])],
+					   [AC_DEFINE([_POSIX_PTHREAD_SEMANTICS], [1],
```

Comment:
That's unfortunate.  Looking at the git log, it was apparently disabled for the strerror_r compatibility with glibc.  I have opened a pull request #82 to re-enable it.